scripts/build_rln_android.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+nwaku_build_dir=$1
+zerokit_dir=$2
+rln_version=$3
+android_arch=$4
+abi=$5
+
+[[ -z "${nwaku_build_dir}" ]] && { echo "No nwaku build directory specified"; exit 1; }
+[[ -z "${zerokit_dir}" ]]     && { echo "No zerokit directory specified";     exit 1; }
+[[ -z "${rln_version}" ]]     && { echo "No rln version specified";           exit 1; }
+[[ -z "${android_arch}" ]]    && { echo "No android architecture specified";  exit 1; }
+[[ -z "${abi}" ]]             && { echo "No abi specified";                   exit 1; }
+
+export RUSTFLAGS="-Ccodegen-units=1"
+
+cargo install cross --git https://github.com/cross-rs/cross
```

Comment:
I initially got the error: 
```
cannot install package `cross 0.2.5`, it requires rustc 1.77.2 or newer, while the currently active rustc version is 1.75.0
```

I see that rustc 1.77.2 is very recent (2024-04-09). 

Maybe we should add to the script to install that specific version, or at least ```rustup upgrade```. For sure safer to do it with a specific version though.

Reply:
Good idea!